seatunnel-dist/src/main/assembly/assembly-bin-ci.xml

```diff
@@ -63,131 +55,122 @@
         </fileSet>
+        <!-- DISCLAIMER -->
         <fileSet>
-            <directory>../seatunnel-connectors/seatunnel-connectors-flink-sql-dist/target/lib</directory>
+            <directory>${basedir}/.././</directory>
             <includes>
-                <include>flink-sql-connector*.jar</include>
+                <include>DISCLAIMER</include>
             </includes>
-            <excludes>
-                <exclude>%regex[.*((javadoc)|(sources))\.jar]</exclude>
-            </excludes>
-            <outputDirectory>/connectors/flink-sql</outputDirectory>
+            <outputDirectory>.</outputDirectory>
         </fileSet>
-        <fileSet>
-            <directory>../seatunnel-connectors/seatunnel-connectors-spark-dist/target/lib</directory>
+    </fileSets>
+
+    <files>
+        <file>
+            <source>../plugin-mapping.properties</source>
+            <outputDirectory>/connectors</outputDirectory>
+        </file>
+    </files>
+    <dependencySets>
+        <!-- ============ Starter Jars ============  -->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <useTransitiveDependencies>true</useTransitiveDependencies>
+            <unpack>false</unpack>
             <includes>
-                <include>seatunnel-connector-spark*.jar</include>
+                <!-- Flink V2 starter -->
+                <include>org.apache.seatunnel:seatunnel-flink-starter:jar</include>
+                <!-- Spark V2 starter -->
+                <include>org.apache.seatunnel:seatunnel-spark-starter:jar</include>
+                <!-- Flink starter -->
+                <include>org.apache.seatunnel:seatunnel-core-flink:jar</include>
+                <!-- Flink SQL starter -->
+                <include>org.apache.seatunnel:seatunnel-core-flink-sql:jar</include>
+                <!-- Spark starter -->
+                <include>org.apache.seatunnel:seatunnel-core-spark:jar</include>
             </includes>
-            <excludes>
-                <exclude>%regex[.*((javadoc)|(sources))\.jar]</exclude>
-            </excludes>
-            <outputDirectory>/connectors/spark</outputDirectory>
-        </fileSet>
-        <fileSet>
-            <directory>../seatunnel-connectors-v2-dist/target/lib</directory>
+            <outputFileNameMapping>${artifact.file.name}</outputFileNameMapping>
+            <outputDirectory>/lib</outputDirectory>
+            <scope>provided</scope>
+        </dependencySet>
+
+        <!-- ============ Connectors Jars ============  -->
+        <!-- SeaTunnel connectors -->
+        <dependencySet>
+            <useProjectArtifact>false</useProjectArtifact>
+            <useTransitiveDependencies>true</useTransitiveDependencies>
+            <unpack>false</unpack>
             <includes>
-                <include>connector-*.jar</include>
+                <include>org.apache.seatunnel:connector-*:jar</include>
             </includes>
```

Comment:
It seems that our CI is strongly dependent on the logic here. Maybe we will modify the CI later to get all the connector jars from connector-dist/target. I remember we no longer release binaries for connectors. Of course, this is something to do later

Reply:
`seatunnel-dist` packaging is to generate tar.gz file, how to do CI needs to be discussed.